src/App.vue

```diff
@@ -156,18 +164,19 @@ export default class App extends Vue {
     this.$router.replace('/').catch(() => {})
 
     // Clear data and display the Tabs page
-    await newRequestModule.resetAnalyzeName()
-    await newRequestModule.mutateName('')
-    await newRequestModule.mutateDisplayedComponent('Tabs')
+    await this.resetAnalyzeName(null)
+    await this.setName('')
+    await this.setDisplayedComponent('Tabs')
   }
 
+  // TODO: Update Pay module
   async onPaymentCancelled () {
-    const { nrId } = newRequestModule
-    const componentName = newRequestModule.displayedComponent
+    const { getNrId } = this
+    const componentName = this.getDisplayedComponent
     // Only do this for New NRs!!!
-    if (nrId && ['SubmissionTabs'].indexOf(componentName) > -1) {
+    if (getNrId && ['SubmissionTabs'].indexOf(componentName) > -1) {
       // Cancel the NR using the rollback endpoint if we were processing a NEW NR
-      await newRequestModule.rollbackNameRequest({ nrId, action: RollbackActions.CANCEL })
+      await this.rollbackNameRequest({ getNrId, action: RollbackActions.CANCEL })
```

Comment:
Check other places where an object is passed to a method... this one is broken because the method expects the first object property to be named `nrId`.

Reply:
I fixed this one but there may be others.